packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx

```diff
@@ -45,34 +76,114 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
   date: {
     fontSize: typography.fontSize.xs,
     color: palette.neutralLight2
+  },
+  unreadTagContainer: {
+    marginVertical: spacing(6),
+    flexGrow: 1,
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  unreadSeparator: {
+    height: 1,
+    backgroundColor: palette.neutralLight5,
+    flexGrow: 1
+  },
+  unreadTag: {
+    color: palette.white,
+    fontSize: typography.fontSize.xxs,
+    fontFamily: typography.fontByWeight.bold,
+    textTransform: 'uppercase',
+    letterSpacing: 0.5,
+    backgroundColor: palette.neutralLight5,
+    paddingHorizontal: spacing(2),
+    paddingVertical: spacing(1),
+    borderRadius: spacing(0.5)
+  },
+  // tailIsAuthor: {
+  //   display: 'flex',
+  //   position: 'absolute',
+  //   right: -7,
+  //   zIndex: -1,
+  //   bottom: 47,
+  //   width: 18,
+  //   height: 12
+  // },
+  // tailOtherUser: {
+  //   display: 'flex',
+  //   position: 'absolute',
+  //   left: -7,
+  //   zIndex: -1,
+  //   bottom: 47,
+  //   width: 18,
+  //   height: 12,
+  //   transform: [{ scaleX: -1 }]
+  // }
+  tailIsAuthor: {
+    display: 'flex',
+    position: 'relative',
+    marginRight: -14,
+    marginTop: -12
+  },
+  tailOtherUser: {
+    display: 'flex',
+    position: 'relative',
+    marginLeft: -14,
+    marginTop: -12,
+    transform: [{ scaleX: -1 }]
```

Comment:
Thoughts on relative vs absolute positioning? Both work, I understand the differences in theory but not sure which to use in practice - seems like both are valid here.

Reply:
Eg. which would be better if the user has zoomed in somehow on their phone, or enlarged text?

Reply:
Personally not a huge fan of the negative margins thing - relative or absolute positioning could work here IMO. I tend to opt for absolute positioning if you know you want to take an object out of the DOM hierarchy, but sometimes it can be trickier to find the correct coordinates with absolute, in which case relative can be useful.

Reply:
it's a case by case basis for me between absolute and relative. e.g. if i know an item, say, a track tile corner tag is supposed to be at the top left, easier to do absolute with top: 0 and left: 0. there are other cases where you care more about the relative positioning. whatever is simplest for the given situation

Reply:
nice thx guys

Reply:
agree with the above, def case by case, and both would work here